data/nmstate/operator.yaml

```diff
@@ -235,12 +239,12 @@ spec:
       name: {{template "handlerPrefix" .}}nmstate-webhook
 ---
 apiVersion: v1
+  annotations:
```

Comment:
This must stay under metadata

Reply:
Something is wrong with rejectOwner patching.